src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/saldo/SaldoUtregning.java

```diff
@@ -46,33 +45,19 @@ public class SaldoUtregning {
     private final LocalDateTime sisteSøknadMottattTidspunktAnnenpart;
     private final SaldoUtregningFlerbarnsdager saldoUtregningFlerbarnsdager;
 
-    SaldoUtregning(Map<Stønadskontotype, Trekkdager> stønadskontoer, // NOSONAR
-                   List<FastsattUttakPeriode> søkersPerioder, List<FastsattUttakPeriode> annenpartsPerioder, SaldoUtregningGrunnlag grunnlag) {
-        this.søkersPerioder = søkersPerioder;
-        this.annenpartsPerioder = fjernOppholdsperioderEtterSisteUttaksdato(søkersPerioder, annenpartsPerioder);
-        this.søkersAktiviteter = grunnlag.getAktiviteter();
-        this.sisteSøknadMottattTidspunktSøker = grunnlag.getSisteSøknadMottattTidspunktSøker().orElse(null);
-        this.sisteSøknadMottattTidspunktAnnenpart = grunnlag.getSisteSøknadMottattTidspunktAnnenpart().orElse(null);
-        this.berørtBehandling = grunnlag.isBerørtBehandling();
-        this.saldoUtregningFlerbarnsdager = new SaldoUtregningFlerbarnsdager(søkersPerioder, this.annenpartsPerioder, søkersAktiviteter,
-            grunnlag.getSpesialkontoTrekkdager(Spesialkontotype.FLERBARN), grunnlag.getSpesialkontoTrekkdager(Spesialkontotype.BARE_FAR_MINSTERETT));
-        this.stønadskonti.putAll(stønadskontoer);
-        Arrays.stream(Spesialkontotype.values()).forEach(k -> this.spesialkonti.put(k, grunnlag.getSpesialkontoTrekkdager(k)));
-    }
-
-
-    SaldoUtregning(Map<Stønadskontotype, Trekkdager> stønadskontoer,
-                   // NOSONAR
-                   List<FastsattUttakPeriode> søkersPerioder,
-                   List<FastsattUttakPeriode> annenpartsPerioder,
-                   boolean berørtBehandling,
-                   Set<AktivitetIdentifikator> søkersAktiviteter,
-                   LocalDateTime sisteSøknadMottattTidspunktSøker,
-                   LocalDateTime sisteSøknadMottattTidspunktAnnenpart,
-                   Trekkdager minsterettDager,
-                   Trekkdager utenAktivitetskravDager,
-                   Trekkdager flerbarnsdager,
-                   Trekkdager farUttakRundtFødselDager) {
+    // NOSONAR
```

Comment:
Nå er det ett felt for hver av spesialkontoene - lett å bomme på rekkefølgen
Ville det vært bedre å sende inn en Map<Spesialkontotype, Trekkdager> ? 
blir en del omskriving av tester - men da kommer det tydelig fram hvilken spesialkonto som settes - nå må man lese det fra posisjon evt fra N dedikerte factories.

Reply:
Bare 2 steder å skrive om, så det skal gå greit